src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Repeater/Given_ItemsRepeater.cs

```diff
@@ -525,6 +525,43 @@ public async Task When_UnloadReload_Then_MaterializeItemsForCurrentViewport()
 			LayoutInformation.GetLayoutSlot(sut.MaterializedElements.OrderBy(e => e.DataContext).First()).Y.Should().Be(0, "Item #0 should be at the origin of the IR (negative offset means we are in trouble!)");
 		}
 
+		[TestMethod]
+		[RunsOnUIThread]
+#if __MACOS__
+		[Ignore("Currently fails on macOS, part of #9282 epic")]
+#endif
+		public async Task When_NumberBox_In_Repeater_Then_CanFocus_And_Edit()
+		{
+			var sut = SUT.Create(
+				source: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+				itemTemplate: new DataTemplate(() => new Border
+				{
+					Child = new Grid
+					{
+						Children =
+						{
+							new NumberBox() {Value = 10},
+						}
+					}
+				}),
```

Comment:
`DataTemplate.ctor(Function<object>)` should not be used:
1. win doesnt have such signature
2. it breaks template-parent initialization (may or may not be relevant here) for uno

use parsed xaml instead:
```suggestion
				itemTemplate: XamlHelper.LoadXaml<DataTemplate>("""
					<Border>
						<Grid>
							<NumberBox Value="10" />
						</Grid>
					</Border>
				"""),
```